### ISSUE TITLE
Make documention more concise

### ISSUE THREAD

**Issue by pjhfggij** [pyrevitlabs/pyRevit#2115]
## Is your feature request related to a problem? Please describe

Documentation feels too repetitive, e.g.:
![image](https://github.com/eirannejad/pyRevit/assets/161205138/94423ce8-113d-4f89-aba4-279ebb5b996a)

## Describe the solution you'd like

Less verbose.

## Additional context

I must admit though that the amount of documentation in this project is very helpful though! Especially video demos. However, when I've started to actually use it, it seems hard to parse due to all the unnecessary verbosity. I realize that it's difficult to have both.

**Comment by sanzoghenzo** [comment]
Read it again: to create a command, you need to create a command **bundle**. 

I feel your frustration of not finding the info you need right away, but you are highlighting the wrong points. 
The breadcrumb repeating the title content is something that all the knowledge management sites do, should all the internet get rid of them? Are you really hurt seeing the same text two times?

Can you please point us to other, more concrete, examples of verbosity you found, so that we can fix it?

**Comment by sanzoghenzo** [comment]
the OP didn't get back with more concrete example in almost a year, can we close this?
